doc/classes/Image.xml

```diff
@@ -76,6 +76,7 @@
 			<param index="0" name="mode" type="int" enum="Image.CompressMode" />
 			<param index="1" name="source" type="int" enum="Image.CompressSource" default="0" />
 			<param index="2" name="lossy_quality" type="float" default="0.7" />
+			<param index="3" name="astc_format" type="int" enum="Image.ASTCFormat" default="0" />
 			<description>
 				Compresses the image to use less memory. Can not directly access pixel data while the image is compressed. Returns error if the chosen compression mode is not available. See [enum CompressMode] and [enum CompressSource] constants.
```

Comment:
How does the `compress()` method handle the compress `mode` parameter with regards to `astc_format`? In other words, why aren't 4x4 and 8x8 ASTC `mode` constants, rather than a new parameter? (S3TC/BPTC/ETC and ASTC can't be used at the same time on a given image.)

Reply:
You don't really need the mode because ASTC is a single compression type, RGBA always. It probably handles this internally.